cloudbuild.yaml

```diff
@@ -0,0 +1,68 @@
+timeout: 3600s # 1 hour
```

Comment:
This might be too short for the backend tests, which can take >2 hours on GitHub Actions

Reply:
I'll change it. I picked an arbitrary time for now, since I wasn't sure what tests would end up there.